### DISCUSSION TITLE
Using Project build number in Slack messages

### DISCUSSION THREAD

**Discussion by maks** [codemagic-ci-cd/codemagic-docs#2036] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Hi Cm team.
Currently a workflows `BUILD_NUMBER` is used in the slack notifications that are sent out when using the YAML:
```
slack:
``` 
section in the `publishing` section.

I'm not sure if I'm missing it in the docs, but is there a setting to make it use the `PROJECT_BUILD_NUMBER` instead?

If not, then I guess this is a feature request to have a property on the `slack` section in yaml to allow this. 

Thanks 🙂

**Comment by rafeysup93** [comment]
Hi @maks,

Thank you for bringing this to our attention. You can always customize your Slack messages by following [this](https://docs.codemagic.io/knowledge-others/slack-api-integration/) document.

**Comment by maks** [comment, marked as answer]
Thanks @rafeysup93 , I was hoping to avoid having to do fully customised slack notifications as I'm not an admin of our slack which makes building a slack app much harder and apart from this one small issue that came up as part of moving from WFE to YAML, we are actually very happy with the default Codemagic slack messages. It's only a small thing, but if it could go on the future feature requests list that would be great 🙂
